docs/en/week07/07.md

```diff
@@ -9,7 +9,7 @@ title: Week 7
 
 ## Lecture part B
 
-
+We discussed self-supervised learning, introduced training an Energy-based models, Latent Variable EBM, specifically K-means example. We also introduced Contrastive Methods including DAE, details of BERT, Contrastive Divergence.
```

Comment:
Do you see the red flag in the image below?

<img width="730" alt="Screenshot 2020-03-21 00 41 51" src="https://user-images.githubusercontent.com/2119355/77219524-01f80600-6b0d-11ea-9de6-b1dd1dfb2dfd.png">

Reply:
It needs to be between 40 and 60 words. Plz fix.